cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -451,7 +458,7 @@ def _aqt_compile(
         }
 
         options_dict: Dict[str, Union[float, Dict[str, Union[cirq.Gate, cirq.Operation, None]]]]
-        options_dict = {}
+        options_dict = {} if options is None else options
```

Comment:
let's do something like this:
```suggestion
        options_dict = {}
        if options:
            options_dict.update(options)
```
that way we don't mutate input dictionaries passed by the user

Reply:
Good point, updated.